src/main/java/mate/academy/spring/validation/Password.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.spring.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = PasswordValidator.class)
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Password {
+    String message() default "Password not repeat";
```

Comment:
it can also fail when we have null password, so lets change it to just `Invalid password`